pkg/cli/newapp/newapp.go

```diff
@@ -457,6 +457,9 @@ func (o *AppOptions) RunNewApp() error {
 				installing = append(installing, t)
 			}
 		case *buildv1.BuildConfig:
+			if t.Spec.Strategy.Type == buildv1.JenkinsPipelineBuildStrategyType {
+				fmt.Fprintf(out, "%sNOTICE: the JenkinsPipeline strategy is deprecated; consider using Jenkinsfiles directly on Jenkins or OpenShift Pipelines.\n", indent)
```

Comment:
Log to `o.Action.ErrOut` and make the message similar to https://github.com/openshift/oc/blob/master/pkg/cli/rollout/latest.go#L114

Reply:
@soltysh done in latest push.